NOTICE

```diff
@@ -3,3 +3,9 @@ Copyright 2020 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache-2 license.
+See bundled license files for details.
+ - compress-lzf:1.0.3
```

Comment:
It seems it is not necessary to list the artifacts which follow the Apache License. I did some research based on https://github.com/apache/dubbo/blob/master/NOTICE, https://github.com/apache/incubator-shardingsphere/blob/master/NOTICE and Flink. We only need to list those artifacts that follow the other license(e.g. BSD) and so on. WDYT?

Reply:
That was my understanding as well.. looks like @lresende agrees.. so lets do it like that.. Something to note is, we should not base on other incubating projects.. but recently graduated ones..